Now sends loginType to anonymous users in login_info

mockoocy opened the pull request:
At MAX IV we don't use proposal-based login nor SSO. This PR makes it so, that in the case of user log in appropriate text shows up ("Sign in").

Also did small cleanup to `UserManager.login_info` - now uses early return.

mockoocy committed:
Code cleanup: move to early return style

axelboc commented on ui/src/components/LoginForm/LoginForm.jsx:
```diff
@@ -11,6 +11,14 @@ import { logIn } from '../../actions/login';
 
 const SSO_PATH = '/mxcube/api/v0.1/login/ssologin';
 
+/** */
```
Empty comment here (just remove)

mockoocy replied:
Something must have distracted me when writing a JSDoc there :laughing: 
Added a JSDoc instead of removing the comment.

axelboc replied:
Haha you didn't have to, the name of the function is clear, the code it contains as well and is concise... and it's an internal function, so really not needed in this case :wink: but it's done, so no worries :grin:

axelboc reviewed (approved):
LGTM

mockoocy committed:
Now sends loginType for anonymous users in login_info.

This information is needed to display appropriate sign in message when
using user-based login.
Previously, the same message was displayed for user-type login and
proposal-type one, if SSO wasn't used.

marcus-oscarsson commented:
Thanks :)
